### ISSUE TITLE
Painfully loud, sudden, random "audio flashbangs" that happen whenever they feel

### ISSUE THREAD

**Issue by SalutanisOrkonus** [LMMS/lmms#7098]
### Bug Summary
Sometimes out of nowhere, the program will just play this painfully loud and sudden bang sound (with reverb. without reverb it makes a loud click, at least in the export I tried doing just now), seems absolutely completely random. Can happen while making, and during export. this has happened for as long as I remember.
#### Steps to reproduce
???? I have no clue ill be honest, I'd say CPU or smth but this also has happened on old projects that dont take up much, so something with some effect maybe??? idk. feels like it primarily happens with any reverb, but I seriously have no idea.
#### Expected behavior
i can make a song without being given a heart attack
#### Actual behavior
im having my own normandy
#### Screenshot
not a screenshot but: 
[examp.zip](https://github.com/LMMS/lmms/files/14231096/examp.zip)
it's actually so randomly common on this one project I can't get an export without it

#### Affected LMMS versions
1.3.0-alpha.1.518+gdd53bec31 and every other version I've ever used, I've just been able to get a capture of it this time

**Comment by SalutanisOrkonus** [comment]
obvious from the subject but very big audio warning on the included soundfiles

**Comment by TechnoPorg** [comment]
I can't load your project on my end because I'm missing your specific VST plugins and samples. If it's not too much trouble, would you be able to create a minimum reproducible project that's self-contained? That would just help me see if I can replicate it on my end, and then we can narrow down the bug a bit more.

**Comment by SalutanisOrkonus** [comment]
[Vaporwave City - Copy.zip](https://github.com/LMMS/lmms/files/14255972/Vaporwave.City.-.Copy.zip)
happens _maaybe_ less often in this one but definitely still happens
might happen more often when messing with properties? still definitely happens sometimes if you leave it playing though
This is an annoying thing to pinpoint cuz it happens seemingly when it feels usually

**Comment by TechnoPorg** [comment]
You're right, this is going to be annoying to pinpoint... This is what your project looks like for me (I'm running master @ c205215), and I'm assuming the plugins are all the same now:
![image](https://github.com/LMMS/lmms/assets/69441745/9746691b-3b16-43bb-996e-4a8f26d7e04f)
And yet I've left it playing for 5 minutes, then tried messing with the settings of all the various effects, and I'm still not able to get that bang to happen.

A couple more debugging questions I have:
- What OS are you on?
- What audio backend are you using? You can find that under Edit > Settings like so:
![image](https://github.com/LMMS/lmms/assets/69441745/6dc2dd72-58e4-4ff4-9c20-748c391322cc)
- Are you frequently maxing out your CPU or RAM when working with the file?
Thanks :)

**Comment by SalutanisOrkonus** [comment]
-Windows 10
-SDL (Simple DirectMedia Layer)
-No

**Comment by TechnoPorg** [comment]
On Windows 11 with SDL, I'm still unable to reproduce this. I unfortunately don't have access to a Windows 10 test machine, and even if I did, I suspect the bug is something highly specific to your audio codec and software setup, which will make it a pain to debug.

**Comment by zonkmachine** [comment]
@SalutanisOrkonus I don't know what's causing this but it's obviously something that causes large spikes in the audio. If there is a reverb unit after a spike it will go from a sudden tick to a thunder clap. A workaround is to insert some unit that clips the signal hard in front of the reverb. The lmms Waveshaper Effect has an option to clip the input. It will clip to +/- 1.0 .

**Comment by IKAZstudios** [comment]
I have no idea if this is relavent, but I experienced simmilar heart attacks with sfxr + reverb. See [this] reddit post. Strangely, it's still very unpredictable. (https://www.reddit.com/r/lmms/comments/16as8ak/sfxr_makes_random_loud_explosion_noise/)
